src/engine/Engine.cpp

```diff
@@ -918,19 +918,18 @@ void Engine::fixViolatedPlConstraintIfPossible()
     _tableau->setNonBasicAssignment( fix._variable, fix._value, true );
 }
 
-bool Engine::processInputQuery( InputQuery &inputQuery )
+bool Engine::processInputQuery( const IQuery &inputQuery )
```

Comment:
Making this const so that we know processInputQuery does not change the InputQuery, which will be "re-processed" later during incremental solving.